### DISCUSSION TITLE
STM32F412VETG6 support

### DISCUSSION THREAD

**Discussion by thezim** [stm32duino/Arduino_Core_STM32#2418] [Variants] [answered]
Is there support for the STM32F12VET6?

The following seems to indicate there is.

https://github.com/stm32duino/Arduino_Core_STM32/blob/f9a35fd714b889475e067d8c2c72c551ea3c0cba/variants/STM32F4xx/F412V(E-G)T/boards_entry.txt#L7

However, in the list "board part number" do not see it as a selection.

![image](https://github.com/stm32duino/Arduino_Core_STM32/assets/14023826/809a1047-a2d0-4daf-b10c-28ec635f85d4)

If this issues doesn't belong in this replay please let me know where it should go.

**Comment by fpistm** [comment, marked as answer]
Hi @thezim 
The STM32F412VET is not supported because it is not declared in the boards.txt.
The file you point is generated automatically. We provide all base files to ease board addition.
See here how to declare it: https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29
